app/src/main/res/values/strings.xml

```diff
@@ -598,6 +598,18 @@ Upload your first media by tapping on the add button.</string>
   <string name="delete_helper_ask_reason_copyright_logo">Logo</string>
   <string name="delete_helper_ask_reason_copyright_other">Other</string>
   <string name="delete_helper_ask_alert_set_positive_button_reason">Because it is</string>
+
+  <string name="category_edit_helper_make_edit_toast">Trying to update categories.</string>
+  <string name="category_edit_helper_show_edit_title">Categories are updated</string>
```

Comment:
Based on my interpretation, the notification title would be "Categories are updated: Success" if the categories are successfully updated, and "Categories are updated: Could not add categories" if they are not, right? In that case I think this string should be `Category update`.